testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeDatabricksCreateTableAsSelectCompatibility.java

```diff
@@ -236,7 +241,8 @@ public void testReplaceTableWithSchemaChangeOnCheckpoint()
         try {
             ImmutableList.Builder<QueryAssert.Row> expected = ImmutableList.builder();
             // Write to the table until a checkpoint file is written
-            for (int i = 0; i < 9; i++) {
+            int insertCount = databricksRuntimeVersion.equals(DATABRICKS_113_RUNTIME_VERSION) ? 10 : 9;
```

Comment:
What changed in Databricks `11.3` in regards to checkpoints compare to the previous versions?
Can we add a comment here to clear out eventual misunderstandings?

Reply:
The test object is replacing a table with schema changes on checkpoint, but 11.3 doesn't create it in `CREATE OR REPLACE TABLE` statement. Added `DELTA_LAKE_EXCLUDE_113` instead.